modules/pel/peltool/peltool.py

```diff
@@ -327,6 +412,17 @@ def main():
 
         sys.exit(0)
 
+    if args.list:
+        if inBMC:
+            listOption(BMCpath, config, args.extension, args.delete)
+            sys.exit(0)
+        else:
+            if not os.path.isdir(args.list):
```

Comment:
Suggestion/FYI: we could use https://docs.python.org/3/library/argparse.html#action to validate the user inputs.

Ignore for now.

Reply:
Currently using action as required, for example in [LN372](https://github.com/ibm-openbmc/openpower-pel-parsers/commit/1fec449f71217319e6f78fa73ab5657b6f0273a2#diff-f91c747e8d1a325f39181d35322d5c5d6f146b18af0b10b74dd0657e1da155cbR372). Will improve the use in future.